packages/commonwealth/client/scripts/views/pages/manage_community/upgrade_roles_form.tsx

```diff
@@ -33,58 +33,66 @@ export class UpgradeRolesForm
       );
     });
 
-    const names: string[] = noAdmins.map((role) => {
+    const names = noAdmins.map((role) => {
       const displayName = app.profiles.getProfile(
         role.Address.chain,
         role.Address.address
       ).displayName;
 
       const roletext = role.permission === 'moderator' ? '(moderator)' : '';
 
-      return `${displayName}: ${smartTruncate(
-        role.Address.address,
-        6
+      const fullText = `${displayName} - ${formatAddressShort(
+        role.Address.address
       )} ${roletext}`;
+
+      return { label: fullText, value: fullText };
```

Comment:
This breaks the upgrade roles functionality, which requires an array of string. I've pushed a patch.